app/services/sms_responder.rb

```diff
@@ -1,82 +0,0 @@
-class SmsResponder
-  attr_reader :url, :params, :signature
-
-  JOIN_KEYWORDS = %w[join].freeze
-  HELP_KEYWORDS = %w[help].freeze
-  STOP_KEYWORDS = %w[stop cancel end quit unsubscribe].freeze
-
-  def initialize(url, params, signature)
-    @url = url
-    @params = params
-    @signature = signature
-  end
-
-  def call
-    return invalid_signature_response unless signature_valid?
-    return not_a_valid_keyword_response unless valid_keyword?
-    telephony_response = respond_to_sms
-    FormResponse.new(
-      success: true,
-      errors: {},
-      extra: telephony_response.to_h.merge(extra_analytics_attributes),
-    )
-  end
-
-  def signature_valid?
-    Twilio::Security::RequestValidator.new(
-      AppConfig.env.twilio_auth_token,
-    ).validate(url, params, signature)
-  end
-
-  def valid_keyword?
-    (JOIN_KEYWORDS + HELP_KEYWORDS + STOP_KEYWORDS).include?(message_body)
-  end
-
-  private
-
-  # @return [Telephony::Response]
-  def respond_to_sms
-    if JOIN_KEYWORDS.include?(message_body)
-      Telephony.send_join_keyword_response(to: message_from)
```

Comment:
I think this is the only place the telephony responses get used. Now that they're gone form the IdP we may want to consider also removing them from the telephony gem.